source/tests/system/nirfmxgsm_driver_api_tests.cpp

```diff
@@ -64,6 +72,351 @@ TEST_F(NiRFmxGSMDriverApiTests, Init_Close_Succeeds)
+  EXPECT_NE(0, orfs_fetch_switching_results_array_response.lower_relative_power_size());
+  EXPECT_NE(0, orfs_fetch_switching_results_array_response.lower_relative_power().size());
+  EXPECT_NE(0.0, orfs_fetch_switching_results_array_response.lower_relative_power(0));
+  EXPECT_NE(0, orfs_fetch_switching_results_array_response.upper_relative_power_size());
+  EXPECT_NE(0, orfs_fetch_switching_results_array_response.upper_relative_power().size());
+  EXPECT_NE(0.0, orfs_fetch_switching_results_array_response.upper_relative_power(0));
+  EXPECT_NE(0, orfs_fetch_switching_results_array_response.lower_absolute_power_size());
+  EXPECT_NE(0, orfs_fetch_switching_results_array_response.lower_absolute_power().size());
+  EXPECT_NE(0.0, orfs_fetch_switching_results_array_response.lower_absolute_power(0));
+  EXPECT_NE(0, orfs_fetch_switching_results_array_response.upper_absolute_power_size());
+  EXPECT_NE(0, orfs_fetch_switching_results_array_response.upper_absolute_power().size());
+  EXPECT_NE(0.0, orfs_fetch_switching_results_array_response.upper_absolute_power(0));
+  EXPECT_SUCCESS(session, pvt_fetch_slot_measurement_array_response);
+  EXPECT_NE(0, pvt_fetch_slot_measurement_array_response.slot_average_power_size());
+  EXPECT_NE(0, pvt_fetch_slot_measurement_array_response.slot_average_power().size());
+  EXPECT_NE(0.0, pvt_fetch_slot_measurement_array_response.slot_average_power(0));
+  EXPECT_NE(0, pvt_fetch_slot_measurement_array_response.slot_burst_width_size());
+  EXPECT_NE(0, pvt_fetch_slot_measurement_array_response.slot_burst_width().size());
+  EXPECT_NE(0.0, pvt_fetch_slot_measurement_array_response.slot_burst_width(0));
+  EXPECT_NE(0, pvt_fetch_slot_measurement_array_response.slot_measurement_status_size());
+  EXPECT_NE(0, pvt_fetch_slot_measurement_array_response.slot_measurement_status().size());
+  EXPECT_EQ(0, pvt_fetch_slot_measurement_array_response.slot_measurement_status(0));
+  EXPECT_NE(0, pvt_fetch_slot_measurement_array_response.slot_maximum_power_size());
+  EXPECT_NE(0, pvt_fetch_slot_measurement_array_response.slot_maximum_power().size());
+  EXPECT_NE(0.0, pvt_fetch_slot_measurement_array_response.slot_maximum_power(0));
+  EXPECT_NE(0, pvt_fetch_slot_measurement_array_response.slot_minimum_power_size());
+  EXPECT_NE(0, pvt_fetch_slot_measurement_array_response.slot_minimum_power().size());
+  EXPECT_NE(0.0, pvt_fetch_slot_measurement_array_response.slot_minimum_power(0));
+  EXPECT_NE(0, pvt_fetch_slot_measurement_array_response.slot_burst_threshold_size());
+  EXPECT_NE(0, pvt_fetch_slot_measurement_array_response.slot_burst_threshold().size());
+  EXPECT_NE(0.0, pvt_fetch_slot_measurement_array_response.slot_burst_threshold(0));
+  EXPECT_SUCCESS(session, pvt_fetch_measurement_status_response);
+  EXPECT_EQ(0, pvt_fetch_measurement_status_response.measurement_status());
+}
+
+// NOTE: disabled due to "typedef struct" in example's source code
```

Comment:
Note: You can use GTEST_SKIP in the test body to document when the test is run why it was skipped. We have some conditional ones like the one [here](https://github.com/ni/grpc-device/blob/a720cea9b26f2409c873e47a3fc8ad64b65bdf73/source/tests/unit/converters_tests.cpp#L131) but you could have a non-conditional one here with the reasoning. That way when you run the test or test suite it will show up as skipped instead of passing.

Reply:
OK, I added a GTEST_SKIP to this test.